services/api/src/main/java/org/exoplatform/automatic/translation/api/AutomaticTranslationService.java

```diff
@@ -16,14 +16,21 @@
  */
 package org.exoplatform.automatic.translation.api;
 
+import org.exoplatform.automatic.translation.api.dto.AutomaticTranslationConfiguration;
+
+import java.util.Locale;
 import java.util.Map;
 
 public interface AutomaticTranslationService {
 
   void addConnector(AutomaticTranslationComponentPlugin translationConnector);
   Map<String,AutomaticTranslationComponentPlugin> getConnectors();
+
+  AutomaticTranslationConfiguration getConfiguration();
   String getActiveConnector();
-  boolean setActiveConnector(String name);
-  boolean setApiKey(String connector, String apikey);
-  //String translate(String message, Locale targetLang);
+
+  void setActiveConnector(String name);
+
+  void setApiKey(String connector, String apikey);
+  String translate(String message, Locale targetLang);
```

Comment:
code formatting and Javadoc for API (In Interfaces it's better to add javadoc for all methods with input & output & behavior & checked exceptions thrown )

Reply:
done